internal/cmd/cmd_del.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/dicedb/dice/wire"
+)
+
+var cDEL = &DiceDBCommand{
+	Name:      "DEL",
+	HelpShort: "DEL deletes all the specified keys in args list",
+	Eval:      evalDEL,
+}
+
+func init() {
+	commandRegistry.AddCommand(cDEL)
```

Comment:
@arpitbbhayani It is not specific to this PR but just noticed `init` function and thought I'd mention this -
[You Should Think Twice Before Using the init Function in Go](https://www.bytesizego.com/blog/init-function-golang)
[100 Go Mistakes and How to Avoid Them #3: Misusing init functions](https://medium.com/@liuqiang9596/100-go-mistakes-and-how-to-avoid-them-3-misusing-init-functions-53d371d4500e)

- Debugging gets complex
- Order of execution of `init` functions across packages/files is not predictable just by reading the code
- Can affect test cases or make them flaky

Should we make explicit initialization function instead? that'll register all command in registry? The logic here is not complex enough that requires adding `init` function as it's solution

Reply:
I am okay taking this trade-off.